pkg/types/installconfig.go

```diff
@@ -351,3 +353,13 @@ type BootstrapInPlace struct {
 	// InstallationDisk is the target disk drive for coreos-installer
 	InstallationDisk string `json:"installationDisk"`
 }
+
+func (installConfig *InstallConfig) WorkerMachinePool() (*MachinePool, error) {
```

Comment:
I think that I would rather this function not return an error. It is sufficient to return a nil `*MachinePool` when the worker cannot be found. Although we do not support it currently, it is not inconceivable that at some point in the future we would allow the user to explicitly omit the worker pool.